conjur/controllers/logout_controller.py

```diff
@@ -14,7 +14,7 @@
 
 # Internals
 from conjur.constants import DEFAULT_NETRC_FILE, DEFAULT_CONFIG_FILE
-from conjur.init import ConjurrcData
+from conjur.data_objects.conjurrc_data import ConjurrcData
```

Comment:
maybe just data_object?

Reply:
folder names should be in plural so its data_objects